Backend/Backend/Infrastructure/AppDbContext.cs

```diff
@@ -126,7 +126,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             .HasForeignKey(e => e.UserId)
             .IsRequired();
 
- 
+        
```

Comment:
Not a huge deal, and you can ignore this one for now, but try to include only the files you are changing next time